docs/tutorials/intro/executing_tasks.ipynb

```diff
@@ -0,0 +1,381 @@
+      "             \u001b[0m'model'\u001b[0m: \u001b[0m'FIREFLY-PERIMETER'\u001b[0m,\n",
+      "             \u001b[0m'os_version'\u001b[0m: \u001b[0m'12.1X47-D20.7'\u001b[0m,\n",
+      "             \u001b[0m'serial_number'\u001b[0m: \u001b[0m'bcc9ff721778'\u001b[0m,\n",
+      "             \u001b[0m'uptime'\u001b[0m: \u001b[0m10477\u001b[0m,\n",
+      "             \u001b[0m'vendor'\u001b[0m: \u001b[0m'Juniper'\u001b[0m}\u001b[0m}\u001b[0m\n",
+      "\u001b[0m\n",
+      "\u001b[0m"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "AggregatedResult: print_result"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from brigade.plugins.tasks import networking\n",
+    "\n",
+    "cmh_spines = brg.filter(site=\"bma\", role=\"spine\")\n",
+    "result = cmh_spines.run(networking.napalm_get,\n",
+    "                        getters=[\"facts\"])                \n",
+    "cmh_spines.run(text.print_result,\n",
+    "              num_workers=1,\n",
+    "              data=result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pretty much the same pattern, just different task on different devices.\n",
+    "\n",
+    "## What is a task\n",
+    "\n",
+    "Let's take a look at what a task is. In it's simplest form a task is a function that takes at least a [Task](../../ref/api/task.html#brigade.core.task.Task) object as argument. For instance:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "hi! My name is host1.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is host2.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is spine00.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is spine01.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is leaf00.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is leaf01.cmh and I live in cmh\u001b[0m\n",
+      "\u001b[0mhi! My name is host1.bma and I live in bma\u001b[0m\n",
+      "\u001b[0mhi! My name is host2.bma and I live in bma\u001b[0m\n",
+      "\u001b[0mhi! My name is spine00.bma and I live in bma\u001b[0m\n",
+      "\u001b[0mhi! My name is spine01.bma and I live in bma\u001b[0m\n",
+      "\u001b[0mhi! My name is leaf00.bma and I live in bma\u001b[0m\n",
+      "\u001b[0mhi! My name is leaf01.bma and I live in bma\u001b[0m\n",
+      "\u001b[0m"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "AggregatedResult: hi"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def hi(task):\n",
+    "    print(f\"hi! My name is {task.host.name} and I live in {task.host['site']}\")\n",
+    "    \n",
```

Comment:
Should we use f-strings here? I'm guessing most people won't be running 3.6.

Reply:
I think we should mention python3.6 in the beginning of the tutorial.